mail_tracking/models/ir_mail_server.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import re
+from openerp import models, api, tools
+
+
+class IrMailServer(models.Model):
+    _inherit = "ir.mail_server"
+
+    def _tracking_headers_add(self, tracking_email_id, headers):
+        """Allow other addons to add its own tracking SMTP headers"""
+        headers = headers or {}
+        headers['X-Odoo-Tracking-ID'] = tracking_email_id
```

Comment:
Any idea what happens when there is an Odoo using this module in play on both ends of the email? I'll setup a test case if nobody does, but I'm wondering if there is going to be cross-talk with the tracking IDs.